src/views/Project/index.jsx

```diff
@@ -140,6 +149,12 @@ export default class Project extends Component {
     }
   }
 
+  componentDidMount() {
+    pageCursors.github = {};
+    pageCursors.bzGoodFirst = {};
+    pageCursors.bzMentored = {};
+  }
```

Comment:
The reason why those are global variables is because updateQuery overwrites the "this" keyword to refer to itself, so I cannot access state in that function.